challangeNetflix/.gitignore

```diff
@@ -0,0 +1,131 @@
+# Created by https://www.gitignore.io/api/xcode,macos,swift
```

Comment:
Ja tem um gitignore, na raiz do projeto, pode apagar esse.

Reply:
tem dois na minha pasta, em lugares diferentes, to na duvida de qual eu apago